clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/AbstractStorageEngine.java

```diff
@@ -609,4 +621,14 @@ private void validateStoreName(StoragePartitionConfig storagePartitionConfig) {
               + " doesn't match current store engine: " + getStoreName());
     }
   }
+
+  public CompressionStrategy getCompressionStrategy() {
```

Comment:
may be cache the chunking and strategy as well in a class var and update when SVS is updated?

Reply:
Hmm... what would be the goal? To avoid an AtomicReference lookup?

Reply:
avoid valueOf call. But looks like compression strategy enum already has a static array, so should be fine.